ci_scripts/generate_docs.py

```diff
@@ -35,4 +38,11 @@ def main() -> int:
 
 
 if __name__ == '__main__':
-    sys.exit(main())
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--output_directory",
+        help="Output directory for docs html files.",
+        default=None,
```

Comment:
You could `default=DOCUMENTATION_DEFAULT_OUTPUT_PATH` here, instead of doing a check in the `main()`.

Reply:
Ah yes, that is nicer 👍